pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -158,44 +136,10 @@ func NewInstanceSettings(httpClientProvider httpclient.Provider) datasource.Inst
 			return nil, fmt.Errorf("error creating http client: %w", err)
 		}
 
-		model := datasourceInfo{
-			profile:       jsonData.Profile,
-			region:        jsonData.Region,
-			assumeRoleARN: jsonData.AssumeRoleARN,
-			externalID:    jsonData.ExternalID,
-			endpoint:      jsonData.Endpoint,
-			namespace:     jsonData.Namespace,
-			datasourceID:  settings.ID,
-			HTTPClient:    httpClient,
-		}
-
-		at := awsds.AuthTypeDefault
```

Comment:
Is everything that was removed here somewhere else?

For example where is this AuthType set to default now if necessary?

Reply:
Json props are defined in the aws-sdk. If "arn" is specified, it will be serialized as `0`, which is the default auth type. Added a test case that asserts that behavior.